services/workflows-service/src/storage/get-file-storage-manager.ts

```diff
@@ -0,0 +1,71 @@
+import { diskStorage } from 'multer';
+import { getFileName } from '@/storage/get-file-name';
+import { z } from 'zod';
+import multerS3 from 'multer-s3';
+import { S3Client, S3ClientConfig, GetObjectCommand } from '@aws-sdk/client-s3';
+import * as tmp from 'tmp';
+import * as fs from 'fs';
+
+import { Readable } from 'stream';
+export type TLocalFile = string;
+
+const isS3BucketConfigured = () => {
```

Comment:
I somewhat disagree, 
yeah onliners are not the best, but, they are more flexible.
If I'll have to add more logic to the bucket configuration. the variable instead of method won't look well.